.github/workflows/sanity.yml

```diff
@@ -15,7 +15,9 @@ on:  # yamllint disable-line rule:truthy
 env:
   GO_VERSION: "1.15"
   # Use IMAGE_TAG_BASE and IMAGE_TAG Makefile variables to build various images
-  IMAGE_TAG_BASE: "quay.io/ramendr/ramen"
+  IMAGE_REGISTRY: "quay.io"
+  IMAGE_REPOSITORY: "ramendr"
+  IMAGE_NAME: "ramen"
   IMAGE_TAG: "sanity"
```

Comment:
IMAGE_TAG_BASE is used for various docker load/save actions below. Best way forward would be to define an IMAGe_TAG_BASE here with the added variables for least change in the yaml.

Reply:
Thanks!